src/lib/rest/RestService.cpp

```diff
@@ -759,7 +759,7 @@ static std::string restService(ConnectionInfo* ciP, RestService* serviceV)
   std::string  details = std::string("service '") + ciP->url + "' not recognized";
```

Comment:
Please add an entry in CHANGES_NEXT_RELEASE file about the changes on this PR, referring the issue number you are solving.